### ISSUE TITLE
Session: Challenges face by early academic researchers

### ISSUE THREAD

**Issue by AzamAsanjarani** [ACEMS/ECRretreat2017Nov#9]
We can discuss this issues:

- Supervision of students

- Doubt about academic career

- Lack of support from supervisors

- Best way of starting a fruitful collaboration

- Feeling emptiness after finishing their PhD

- job hunting (both academic and industry)

**Comment by samclifford** [comment]
I'd be keen on this. My uni has an "Effective Supervisory Practices" module designed to help new supervisors. Would be happy to discuss the sort of things we learned in it. Also the potential to turn informal supervision into more formal stuff would be good, because I seem to spend a lot of time helping a few PhD students.

**Comment by dinesharanathunga** [comment]
I agree, this session would provide insight in to several important issues for an ECR

**Comment by samclifford** [comment]
Could possible roll #1 into this.
